tensorflow/lite/c/c_api_internal.h

```diff
@@ -279,7 +279,7 @@ typedef enum {
 // The delegates should use zero or positive integers to represent handles.
 // -1 is reserved from unallocated status.
 typedef int TfLiteBufferHandle;
-const TfLiteBufferHandle kTfLiteNullBufferHandle = -1;
+#define kTfLiteNullBufferHandle ((TfLiteBufferHandle)(-1))
```

Comment:
how about just kTfLiteNullBufferHandle (-1)

Seems like the type conversion is not necessary here.

Reply:
I'm okay. will fix in later. (But I can't find necessity of `TfLiteBufferHandle` now).